testsuite/features/reposync/srv_check_installer_updates.feature

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Check Installer Update repositories
+  In order to use installer updates
+  As root user
+  I want to check that Installer-Updates Channels got synced when adding the product
```

Comment:
Same reason
```
When I add the product
I want the installer update channels to be synched
```

Reply:
I do not get this. The test just verify that the installer update repo got synced.
The product is not added here.
I could understand to move the test into the feature where also the sync happens, but here we really just check that
the repositories got enabled.

Reply:
It's only about labeling. A "cucumber feature" is a feature of the product. Not a "test feature". Just avoid the obvious "check", "verify", "test" and say what the product is supposed to do.